command/pr_create.go

```diff
@@ -196,8 +193,15 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 
 		fmt.Fprintln(cmd.OutOrStdout(), pr.URL)
 	} else if action == PreviewAction {
+		var title, body string
+		if title != "" {
```

Comment:
This part is actually for consistency, we don't wan `body=&title=` if there is no body and title at all.